kornia/losses/ms_ssim_l1.py

```diff
@@ -0,0 +1,150 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# Based on:
+# https://github.com/psyrocloud/MS-SSIM_L1_LOSS
+
+
+class MS_SSIM_L1Loss(nn.Module):
+    r"""Creates a criterion that computes MS-SSIM + L1 loss.
+
+    According to [1], we compute the MS-SSIM + L1 loss as follows:
+
+    .. math::
+        \text{loss}(x, y) = \alpha \cdot \mathcal{L_{MS-SSIM}}(x,y)+(1 - \alpha) \cdot G_\alpha \cdot \mathcal{L_1}(x,y)
+
+    Where:
+        - :math:`\alpha` is the weight parameter.
+        - :math:`x` and :math:`y` are the reconstructed and true reference images.
+        - :math:`\mathcal{L_{MS-SSIM}}` is the MS-SSIM loss.
+        - :math:`G_\alpha` is the sigma values for computing multi-scale SSIM.
+        - :math:`\mathcal{L_1}` is the L1 loss.
+
+    Reference:
+        [1]: https://research.nvidia.com/sites/default/files/pubs/2017-03_Loss-Functions-for/NN_ImgProc.pdf#page11
+
+    Args:
+        sigmas: gaussian sigma values.
+        data_range: the range of the images.
+        K: k values.
+        alpha : specifies the alpha value
+        compensation: specifies the scaling coefficient.
+
+    Returns:
+        The computed loss.
+
+    Shape:
+        - Input: :math:`(N, C, H, W)`.
+        - Output: :math:`(N,)` or scalar.
+
+    Examples:
+        >>> input1 = torch.rand(1, 3, 5, 5)
+        >>> input2 = torch.rand(1, 3, 5, 5)
+        >>> criterion = kornia.losses.MS_SSIM_L1Loss()
+        >>> loss = criterion(input1.cuda(0), input2.cuda(0))
```

Comment:
Better remove cuda. We want it to be minimal for doctests.

Reply:
done